tools/meteor-services/deploy.js

```diff
@@ -144,7 +144,8 @@ var authedRpc = function (options) {
   var infoResult = deployRpc({
     operation: 'info',
     site: rpcOptions.site,
-    expectPayload: []
+    expectPayload: [],
+    qs: options.qs
```

Comment:
Note that this means all query strings are passed through to all `/info` requests (not just on deploy). I'm not sure if this is likely to cause problems..